src/components/progress-report/community-stories/progress-report-community-story.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@nestjs/common';
+import { UnsecuredDto } from '~/common';
+import {
+  withEffectiveSensitivity,
+  withScope,
+} from '../../authorization/policies/conditions';
+import { Prompt } from '../../prompts/dto';
+import { PromptVariantResponseListService } from '../../prompts/prompt-variant-response.service';
+import { ProgressReport } from '../dto';
+import { ProgressReportHighlight as Highlight } from '../dto/hightlights.dto';
```

Comment:
Kind of the same idea, if we're using something in `highlights` and `communityStories`, might be a good idea to pull it out of highlights and make it generic for both to use

Reply:
This is a copy paste error.